### PR TITLE
Migrate KT to /tutorials

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

Migrate KT path to `/tutorials`

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/migrate_kt

### Validation

Note that any validation of this change needs to ensure that the following build environments continue to work:

- local
- staging here in KT with Semaphore
- staging in CD
- live on CD
- old site redirects to CD

### CODE REVIEW

**Comment by ybyzek** [comment]
@hartantothio I don't know much about `robots.txt` and `sitemap.xml` -- do those need to be updated as well?

**Comment by ybyzek** [comment]
Current PR has some pass and some fail scenarios:

Works:

* Local build
* Developer site

Does not work:

* In staging:  on index.html, links to individual KTs don’t work, e.g.
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/branch_name/ — KT link assumes `/tutorials`, not `branch_name`
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/branch_name/kafka-console-consumer-producer-basics/confluent.html

cc: @colinhicks (as we are discussing in Slack), @hartantothio 

Fix may be to use Jekyll to dynamically build the `index.html` file

**Comment by ybyzek** [comment]
Worked through this with @colinhicks (thank you!!). I think the latest commit addresses the issue above, in that instead of hardcoding `/tutorials`, it dynamically builds the link based off relative_url

**Comment by ybyzek** [comment]
Folks I think this PR is ready for review/approval. Please look at it from all angles, feedback welcome, thank you!

File: .semaphore/live-site-deploy.yml [hunk 1]
```diff
@@ -20,5 +20,5 @@ blocks:
             - npm install
             - gem install bundler
             - bundle install
-            - JEKYLL_ENV=production bundle exec jekyll build
-            - aws s3 cp --recursive ./_site s3://kafka-tutorials.confluent.io/
+            - JEKYLL_ENV=production bundle exec jekyll build --baseurl /tutorials
```
Comment (bbejeck) [inline .semaphore/live-site-deploy.yml:23]: Should the `pr-staging-deploy.yml` and `staging-site-deploy.yml` files get updated as well?

**Review by bbejeck — COMMENTED** [review]
@ybyzek thanks for the PR.  I'll have time for a more thorough review later, but I made a quick pass.

**Comment by ybyzek** [comment]
Closing PR to open a new one against latest mater

**Pull request closed without merging**